src/api/routes/series.js

```diff
@@ -227,4 +230,93 @@ router.get('/:bbid/relationships',
 		return res.status(200).send(seriesRelationshipList);
 	});
 
+/**
+ *	@swagger
+ * '/series':
+ *   get:
+ *     tags:
+ *       - Browse Requests
+ *     summary: Gets a list of Series related to another Entity
+ *     description: BBID of an Author or an Edition or an EditionGroup or a Publisher or a Work is passed as query parameter and it's Series are fetched
+ *     operationId: getRelatedSeriesByBbid
+ *     parameters:
+ *       - name: author
+ *         in: query
+ *         description: BBID of the corresponding Author
+ *         required: false
+ *         schema:
+ *           type: string
+ *           format: uuid
+ *       - name: work
+ *         in: query
+ *         description: BBID of the corresponding Work
+ *         required: false
+ *         schema:
+ *           type: string
+ *           format: uuid
+ *       - name: edition
+ *         in: query
+ *         description: BBID of the corresponding Edition
+ *         required: false
+ *         schema:
+ *           type: string
+ *           format: uuid
+ *       - name: edition-group
+ *         in: query
+ *         description: BBID of the corresponding Edition Group
+ *         required: false
+ *         schema:
+ *           type: string
+ *           format: uuid
+ *       - name: publisher
+ *         in: query
+ *         description: BBID of the corresponding Publisher
+ *         required: false
+ *         schema:
+ *           type: string
+ *           format: uuid
+ *       - name: seriesOrderingType
+ *         in: query
+ *         description: filter by Series Ordering Type
+ *         required: false
+ *         schema:
+ *           type: string
+ *           enum: [Automatic, Manual]
```

Comment:
What use cases did you have in mind or this?
I'm not really convinced is would be particularly useful, but I also don't think it hurts in any way to have it there.
I would also propose shortening it to `orderingType` to match what we do for other entities.

Reply:
Not really I can think of a scenario where it would be useful but yeah, there isn't any harm to have it there.